### ISSUE TITLE
Icons lack alt text

### ISSUE THREAD

**Issue by denyspoltorak** [alex-shpak/hugo-book#772]
[Nu Html Checker](https://validator.w3.org/nu/) shows lots of errors:
An img element must have an alt attribute, except under certain conditions. For details, consult [guidance on providing text alternatives for images](https://www.w3.org/WAI/tutorials/images/).
`<img src=/icons/chevron-right.svg class=book-icon>`

I've fixed that by adding `alt="+"` to the chevron code of menu-filetree.html, but the issue may happen with the book logo and translations, thus it may be better to fix in the icon partial, if you thick that the issue is serious.

**Comment by alex-shpak** [comment]
Hello!
I pushed a set of changes regarding accessibility, it includes translatable alts everywhere 
https://github.com/alex-shpak/hugo-book/commit/6baf098e541dc542ad0b06efbfc889658eda62d7

**Comment by denyspoltorak** [comment]
Thank you!
I'll have to cherry-pick it later.

**Comment by denyspoltorak** [comment]
Hello!
There is another issue reported by [PageSpeed Insights](https://pagespeed.web.dev/) for landing pages, including the one of the latest Hugo Book demo:

Form elements do not have associated labels
Labels ensure that form controls are announced properly by assistive technologies, like screen readers. Learn more about form element labels.
Failing Elements:
- body.book-kind-home > input#menu-control `<input type="checkbox" class="hidden toggle" id="menu-control">`
- body.book-kind-home > input#toc-control `<input type="checkbox" class="hidden toggle" id="toc-control">`

**Comment by denyspoltorak** [comment]
Seems to be fixed by wrapping the inputs and label in baseof.html with `{{if ne .Layout "landing"}}` and `{{end}}`

**Comment by alex-shpak** [comment]
I don't think it should be a problem, those are virtual triggers to show/hide menu and ToC in mobile view, and they made to be invisible and transparent, so screen readers should not stumble on them.

**Comment by denyspoltorak** [comment]
That still may affect page rank

**Comment by alex-shpak** [comment]
in ae68894a881db1ed2ffd92eab999b4e959d375d9 checkboxes have been moved to `header.html` template, should be fixed.
